src/Spaces/ConstantSpace.jl

```diff
@@ -254,7 +254,7 @@ choosedomainspace(M::ConcreteMultiplication{D,UnsetSpace},sp::Space) where {D<:C
 Base.isfinite(f::Fun{CS}) where {CS<:ConstantSpace} = isfinite(Number(f))
 
 
+## Calculus
 
-
-
-
+integrate(f::Fun{<:ConstantSpace{<:IntervalOrSegment}}) = Number(f) * Fun(Chebyshev(domain(f)))
```

Comment:
`Chebyshev` isn't defined in this package, so perhaps you want `Fun(domain(f))`

Reply:
Oh yes, thanks! I've update the code.